### PR TITLE
Reader: migration of "Tags" store from flux to redux

### PR DESCRIPTION
**Pull request by TooTallNate**
**Status:** work-in-progress

A lot of this is cargo-culted from the "lists" Redux code at this point, since it's a lot of the same interactions.

Still need to finish writing reducers and actions. I think the selectors are good to go.

Part of fixing #6607.

Test live: https://calypso.live/?branch=update/migrate-reader-tags-flux-redux

### CODE REVIEW

File: client/state/reader/tags/selectors.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
+/*
+ * External dependencies
+ */
+import filter from 'lodash/filter';
```
Comment (bluefuton) [inline client/state/reader/tags/selectors.js]: You can now do

`import { filter, includes, sortBy, find } from 'lodash'`

instead to neaten this up :)

**Comment by blowery** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;vQRezRhL&#x2F;10-port-reader-tags"> Port reader-tags</a>

**Comment by lancewillett** [comment]
Closing as this was a proof-of-concept and looks abandoned. Feel free to re-open later if it comes back up as important or necessary (or you have time).

**Comment by blowery** [comment]
moved to https://github.com/Automattic/wp-calypso/pull/8589

**Pull request closed without merging**